assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -107,6 +107,10 @@ header[role="banner"] {
   height: 14px;
 }
 
+.countryGroups summary {
```

Comment:
Can we double-check this works in Safari? I discovered some styling issues with CSS targeting the `summary` element but I fixed it quickly without much detailed analysis and some of my assumptions might have been wrong: https://github.com/guardian/support-frontend/pull/973/files

Reply:
Yep, tested in Safari, all clear 👌